packages/amplify-util-mock/src/CFNParser/resource-processors/iam-resource-process.ts

```diff
@@ -0,0 +1,19 @@
+import { CloudFormationParseContext } from "../types";
+
+export function iamPolicyResourceHandler(resourceName, resource, cfnContext: CloudFormationParseContext, transformResult: any) {
+  const processedResource = {
+    cfnExposedAttributes: {},
+    Ref: `IAMPolicy${resource.Properties.PolicyName}`
+  }
+  return processedResource;
+}
+
+export function iamRoleResourceHandler(resourceName, resource, cfnContext: CloudFormationParseContext, transformResult: any) {
+  const processedResource = {
+    cfnExposedAttributes: {'Arn': 'Arn', RoleId: 'RoleId'},
+    Ref: `IAMRole${resource.Properties.RoleName}`,
+    Arn: 'IAM-ARN',
+    RoleId: `AIDAJQABLZS4A3QD${Math.floor(Math.random()*100)}Q`
```

Comment:
It would be nice to have this mock account id specified at a single place even if it is not used yet.